external-crates/move/crates/move-analyzer/src/completion.rs

```diff
@@ -194,50 +195,71 @@ fn get_cursor_token(buffer: &str, position: &Position) -> Option<Tok> {
     }
 }
 
-/// Handle context-specific auto-completion requests with lbrace (`{`) trigger character.
```

Comment:
This is an unrelated refactoring that I sneaked in to get rid of `typed_ast` in `Symbols` (it was annoying that it was only used in this one place and it wasn't even needed as the same info is encoded elsewhere)